modules/user_data/templates/install_graphdb.sh.tpl

```diff
@@ -7,26 +7,11 @@ until ping -c 1 google.com &> /dev/null; do
   sleep 5
 done
 
-timedatectl set-timezone UTC
-
-# install tools
-
-apt-get update
-apt-get -o DPkg::Lock::Timeout=300 install -y unzip jq nvme-cli openjdk-11-jdk bash-completion
-
 curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
Huh, we could bundle this with Packer. However.. we gotta handle different architectures (arm i.e.) meaing more AMIs. Let's discuss it..

Reply:
yeah, we spoke with Viktor about this eventually will be moved in the Packer. We decided to leave it like this now until we know what AMIs we want to have exactly.

Reply:
Sounds reasonable